src/sdk/sql_cluster_router.cc

```diff
@@ -3282,20 +3343,21 @@ hybridse::sdk::Status SQLClusterRouter::HandleLongWindows(
     return {};
 }
 
-bool SQLClusterRouter::CheckPreAggrTableExist(const std::string& base_table, const std::string& base_db,
-                                              const openmldb::base::LongWindowInfo& lw) {
+bool SQLClusterRouter::CheckPreAggrTableExist(const std::string& base_table,
+                                              const std::string& base_db,
+                                              const openmldb::base::LongWindowInfo& lw,
+                                              ::hybridse::sdk::Status* status) {
     std::string meta_db = openmldb::nameserver::INTERNAL_DB;
     std::string meta_table = openmldb::nameserver::PRE_AGG_META_NAME;
     std::string filter_cond = lw.filter_col_.empty() ? "" : " and filter_col = '" + lw.filter_col_ + "'";
     std::string meta_info = absl::StrCat(
         "base_db = '", base_db, "' and base_table = '", base_table, "' and aggr_func = '", lw.aggr_func_,
         "' and aggr_col = '", lw.aggr_col_, "' and partition_cols = '", lw.partition_col_,
-        "' and order_by_col = '", lw.order_col_, filter_cond);
+        "' and order_by_col = '", lw.order_col_, "'", filter_cond);
     std::string select_sql =
-        absl::StrCat("select bucket_size from ", meta_db, ".", meta_table, " where ", meta_info, "';");
-    hybridse::sdk::Status status;
-    auto rs = ExecuteSQL("", select_sql, &status);
-    if (!status.IsOK()) {
+        absl::StrCat("select bucket_size from ", meta_db, ".", meta_table, " where ", meta_info, ";");
+    auto rs = ExecuteSQL("", select_sql, status);
```

Comment:
you can also use `cluster_sdk_->GetTableInfo` to check if table is exist

Reply:
We don't just check for table existence by `table name`, [Here](https://github.com/4paradigm/OpenMLDB/pull/1334#discussion_r816457296) is the previous discussion.

Reply:
if not only check table exist, the function name `CheckPreAggrTableExist` has some confusing